paddle/fluid/operators/activation_op.cc

```diff
@@ -596,6 +596,13 @@ class SwishOpMaker : public framework::OpProtoAndCheckerMaker {
     AddInput("X", "Input of Swish operator");
     AddOutput("Out", "Output of Swish operator");
     AddAttr<float>("beta", "Constant beta of swish operator").SetDefault(1.0f);
+    AddAttr<bool>("use_mkldnn",
+                  "(bool, default false) Only used in mkldnn kernel")
+        .SetDefault(false);
+    AddAttr<bool>("is_test",
+                  "(bool, default false) Set to true for inference only, false "
+                  "for training. Some layers may run faster when this is true.")
+        .SetDefault(false);
```

Comment:
Are the attributes `use_mkldnn` and `is_test` needed? I don't see `is_test` used, and `use_mkldnn` only used in unit-tests.

Reply:
Could you remove these two attributes in the next PR?

Reply:
Without `use_mkldnn` if you disableconv_swish_mkldnn_fuse_pass then swish is not running on mkldnn so I think it's better that it works both with and without fuse. For `is_test` I'm still investigating.

Reply:
> if you disable conv_swish_mkldnn_fuse_pass then swish is not running on mkldnn

We can use `mkldnn_placement_pass` to enable activation to run on MKLDNN. This pass is default ON now.

Reply:
`mkldnn_placement_pass` turns existing `use_mkldnn` attributes to `true` in operators that have the attribute already there. If I remove `use_mkldnn` attribute then `mkldnn_placement_pass` doesn't turn it on.
I did try to remove it to test if it was necessary. After I removed it and run shufflenet by "Test command" I provided DNNL (MKLDNN) was not running the eltwise_swish algorithm. I've seen it by setting env flag `DNNL_VERBOSE=1`.

Reply:
> Without `use_mkldnn` if you disable conv_swish_mkldnn_fuse_pass then swish is not running on mkldnn so I think it's better that it works both with and without fuse pass on mkldnn. For `is_test` I'm still investigating.

I see that **is_test** attribute is not tested in activation unit-tests. Could you add the **is_test** attribute tests in the next PR?

Reply:
@GaoWei8 is_test , currently is not used, but it may be used for in-place support of mkl-dnn operators (including activations) . I'm working on this in-place support. Once we know if we need is_test for activation's in-place then we either add unit tests or remove is_test (if not needed).

Reply:
@jczaja 
- Since `is_test` is not used now, Could you remove it at first for [OP Input Output Attribute Compatibility Modification](https://github.com/PaddlePaddle/Paddle/wiki/OP-Input-Output-Attribute-Compatibility-Modification(English-Version))? Adding attribute is easy, but removing attribute will cause the Compatibility problem.
- If you want to add it for activation's in-place, you could add it back later.

Reply:
@luotao1  Ok, we will proceed with removing is_test from activations.